### ISSUE TITLE
403 / Forbidden

### ISSUE THREAD

**Issue by schroejahr2** [markshust/docker-magento#241]
Hi, im still trying to set up the containers. But i struggle at the install.
I have seen issue #121 but even the result is the same, the issue is another.

I think it is an authentication error. I know  i need user / pass. where can i change that ? 
I have put the pub / priv key into the auth.json. Do i need to do anything else ? 
I read the page, maybe i missed it.

Thank you !

```
Package operations: 415 installs, 0 updates, 0 removals
  - Installing magento/framework (102.0.4): Downloading (connecting...)

  [Composer\Downloader\TransportException]
  The 'https://repo.magento.com/archives/magento/framework/magento-framework-102.0.4.0.zip' URL could not be accessed: HTTP/1.1 403 Forbidden
```

**Comment by markshust** [comment]
- What host operating system are you running?
- What method are you using for installation (automated, manual, existing, ?)?
- What are the commands you are running?

This doesn't have anything to do with Composer/auth keys/etc. Something is going on with how the code is executing on a permission-level within the containers.

**Comment by schroejahr2** [comment]
- Im using ubuntu 16.04 LTS
- i tried automated and manual, not existing. have tried multiple times in a new folder.
- from the readme. Version 2.3.4. Changed ports in docker-compose as follows: 

`ports:
      - "8088:8000"
      - "8443:8443"`

**Comment by markshust** [comment]
@schroejahr2 sorry in the delay of responding, just noticed your reply now.

how do you have the host bind mounts setup in your `docker-compose.dev.yml` file? see following screenshot:
![Screen Shot 2020-01-31 at 1 17 21 PM (1)](https://user-images.githubusercontent.com/437029/78391175-dd436a00-75b3-11ea-9cbb-bee0fbc92854.png)

FYI if you join the m.academy slack group there is a `#m2-docker-dev-env` channel where you can probably get help a lot quicker. invite: https://join.slack.com/t/mdotacademy/shared_invite/zt-99gglpd0-Flij6zPLqsL3DB5jcuXTzQ

**Comment by schroejahr2** [comment]
Hi, thank you for the reply. The error has been fixed. 

For someone who has the same problem: 

A few weeks before i tried the first install, and the login data was stored somewhere in the home directory in ".composer" or something. those user data was erronous, which resulted in the "forbidden" message in the installs a few weeeks later.

I removed the file or the containing folder and everything works well.

Thank you very much.
